### PR TITLE
Panel: header and footer now reflow properly when zoomed in

### PR DESCRIPTION
**Pull request by TristanWatanabe**
#### Pull request checklist

- [x] Addresses an existing issue: [Bug 12294](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/12294), #10905 
- [x] Include a change request file using `$ yarn change`

#### Description of changes
- Moved navigation, header and footer inside scrollable content div.
- Added height based media queries that 'fixes' header and footer to panel when current viewport height meets the minimum set.
- Since navigation, header and footer are now inside the scrollable content - when user zooms in and the viewport height no longer meets the minimum, the navigation/header and footer are 'unfixed' and becomes part of the scrollable content.

Panel Reflow (2)

### CODE REVIEW

File: packages/react/src/components/Panel/Panel.styles.ts [hunk 1]
```diff
@@ -258,7 +258,14 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
     commands: [
       classNames.commands,
       {
-        marginTop: 18,
+        paddingTop: 18,
+        selectors: {
+          [`@media (min-height: ${ScreenWidthMinMedium}px)`]: {
```
[the remark this answers is not in the document]
Comment (TristanWatanabe) [reply packages/react/src/components/Panel/Panel.styles.ts:263]: We don't have any height based constants in the repo and if I did add one, it ultimately would have been the same value as `ScreenWidthMinMedium` which seemed redundant 😅

**Comment by chadsowald** [comment]
@TristanWatanabe Any chance this will be back-ported to Fluent v7?

**Comment by jacksonv1lle** [comment]
This PR has broken several of our custom Panel components 😩.

**Comment by ecraig12345** [comment]
@jacksonv1lle If you file an issue with an isolated repro, our shield person will look into it. You can start from "Export to CodePen" on one of the [Panel examples](https://developer.microsoft.com/en-us/fluentui#/controls/web/panel).

**Comment by jacksonv1lle** [comment]
@ecraig12345 That's OK. We have decided to just update the custom styles to comply with the new Panel structure.

**Pull request merged**